src/js/selection.js

```diff
@@ -1,18 +1,15 @@
-/*global Util */
-var Selection;
-
 (function () {
     'use strict';
 
     function filterOnlyParentElements(node) {
-        if (Util.isBlockContainer(node)) {
+        if (MediumEditor.util.isBlockContainer(node)) {
             return NodeFilter.FILTER_ACCEPT;
         } else {
             return NodeFilter.FILTER_SKIP;
         }
     }
 
-    Selection = {
+    var Selection = {
```

Comment:
In fact it was to keep consistency with other files.
For example, you defined:

```js
MediumEditor.extensions.paste = MediumEditor.Extension.extend({
```

Why don't you do?

```js
var PasteHandler = MediumEditor.Extension.extend({
    // ...
});

MediumEditor.extensions.paste = PasteHandler;
```